app.py

```diff
@@ -58,9 +58,8 @@ def login_user():
     req=request.get_json()
     user=User.query.filter_by(email=req['email']).first()
     if user and bcrypt.check_password_hash(user.password, req['password']):
-        profile=UserProfile.query.filter_by(user_id=user.id).first()
-        print(profile)
-        return jsonify(profile.serialize())
+        profile=UserProfile.query.filter_by(user_id=user.id).first().serialize()
+        return jsonify({**profile, 'email':user.email})
```

Comment:
What are these double asterisks for? Are we expecting to pass in multiple profile objects?

Reply:
This is called *splatting* in Python. I am 'spreading' the user profile object and then adding the email from the user we found and sending it all back in one response.

Reply:
Nice! Thanks for the explanation